### PR TITLE
use ApexCharts instead of HighCharts

### PR DESCRIPTION
**Pull request by vol1ura**
Highcharts license is not free, and there are problems (cors) with loading of the library, so I suggest using Apexcharts, and adding js file to assets as other js.

There is a small difference in the set of options, I tried to keep the previous style of charts.

image

image

### CODE REVIEW

File: app/views/rails_performance/layouts/rails_performance.html.erb [hunk 1]
```diff
@@ -6,9 +6,6 @@
     <title>Rails Performance</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag if ::Rails::VERSION::STRING.to_f >= 5.2 %>
-    <script>
-      window.APP_TIME_ZONE = '<%= Rails.application.config.time_zone.presence || 'UTC' %>';
```
Comment (igorkasyanchuk) [inline app/views/rails_performance/layouts/rails_performance.html.erb:10]: can we keep it and implement something like it?

Comment (igorkasyanchuk) [reply app/views/rails_performance/layouts/rails_performance.html.erb:10]: @vol1ura maybe some other options can be considered

Comment (vol1ura) [reply app/views/rails_performance/layouts/rails_performance.html.erb:10]: Here I try to use option `datetimeUTC: false` in charts options. It seems the time is displayed correctly now. Please, check

Comment (igorkasyanchuk) [reply app/views/rails_performance/layouts/rails_performance.html.erb:10]: hmm, maybe it works fine

could you please check if you have the same. I just see some inconsistency and don't know why

image

Comment (igorkasyanchuk) [reply app/views/rails_performance/layouts/rails_performance.html.erb:10]: also not sure about this part of the tooltip, I think it's obvious anyway

image


also not sure but I have a feeling this chart library works slower on mouse hover

but I like the tooltip with time on X-axis

Comment (igorkasyanchuk) [reply app/views/rails_performance/layouts/rails_performance.html.erb:10]: another example of chart library that can be used: https://echarts.apache.org/examples/en/index.html

I used it few times and it's quite powerful

Comment (vol1ura) [reply app/views/rails_performance/layouts/rails_performance.html.erb:10]: > also not sure about this part of the tooltip, I think it's obvious anyway

Tooltip title can be disabled on all charts, I'll add this param

Comment (igorkasyanchuk) [reply app/views/rails_performance/layouts/rails_performance.html.erb:10]: @vol1ura please also move Y axis to the left side. Basically, make a chart very similar to the current version.

After that, I think I can try to release "alpha" version and try it on production

**Comment by igorkasyanchuk** [comment]
PR merged, I think it works fine
thanks!

**Pull request merged**